cmd/clusterctl/clusterdeployer/bootstrap/minikube/minikube.go

```diff
@@ -94,7 +96,8 @@ func (m *Minikube) Create() error {
 }
 
 func (m *Minikube) Delete() error {
-	_, err := m.exec("delete")
+	args := []string{"delete", "--profile=" + minikubeClusterNamePrefix + util.RandomString(5)}
```

Comment:
Same here as above, this breaks the workflow given that we're going to be trying to delete a cluster with a different name

Reply:
In this case, I believe you are correct, since the profile name is generated at delete time separately from the one created when the bootstrapper is initialized.